tensorflow/lite/g3doc/models/convert/metadata_writer_tutorial.ipynb

```diff
@@ -223,7 +223,8 @@
         "    writer_utils.load_file(_MODEL_PATH), [_INPUT_NORM_MEAN], [_INPUT_NORM_STD],\n",
         "    [_LABEL_FILE])\n",
         "\n",
-        "# Verify the metadata generated by metadata writer.\n",
+        "# Verify the metadata generated by the 
+metadata writer.\n",
```

Comment:
It looks like there is an inadvertent new line inserted here, perhaps?

Reply:
Suggested changes are done. Thank You